lib/moon_web/examples/design/switch_example/using_with_forms.ex

```diff
@@ -0,0 +1,38 @@
+defmodule MoonWeb.Examples.Design.SwitchExample.UsingWithForms do
+  @moduledoc false
+
+  use Moon.StatelessComponent
+  use MoonWeb, :example
+
+  alias Moon.Design.Form.Switch
+  alias Moon.Components.Form
+  alias MoonWeb.Pages.Tutorials.AddDataUsingForm.User
+
+  prop(user_changeset, :any, default: User.changeset(%User{}))
+
+  def render(assigns) do
+    ~F"""
+    <div>
+      <Form for={@user_changeset}>
+        <Switch id="switch_11" />
```

Comment:
name and value should be set here. name should be a required property for Moon.Design.Form.Switch

Reply:
Missing attributes added